### ISSUE TITLE
Cannot use selectors containing attribute

### ISSUE THREAD

**Issue by yamashita-tutorial-tomoya** [fczbkk/css-selector-generator#262]
An invalid selector string containing attribute is created for the following elements.

```html
// MyElement
<input type="checkbox" value="9659985">
```

The effect of CSS.escape() will return the following string for elements where value is a number.
It cannot be queried with document.querySelector and returns null.

```javascript
getCssSelector(MyElement)
// => [value=''\\39 659985''] 
```

I can make the PR if your want to.

Regards

**Comment by fczbkk** [comment]
Nice catch @yamashita-tutorial-tomoya.

According to specification, attribute selector value has to be `ident-token`. That means it can not start with a number.
https://drafts.csswg.org/css-syntax-3/#typedef-ident-token

I'd say that the value of an `<input />` element should not be used in attribute selector. And I think same should go for `<option />`. I will add a condition that will reject them as candidates.

**Comment by fczbkk** [comment]
Released `v3.6.1` which should fix this issue.
https://www.npmjs.com/package/css-selector-generator/v/3.6.1

**Comment by fregante** [comment]
I don’t think this is the correct fix, `[value="9659985"]` works. I think `CSS.escape` should not be run on attribute selectors, I think you only need to escape the specific quote you're using to wrap the value.
